src/Localization/.gitignore

```diff
@@ -0,0 +1,3 @@
+# https://lingui.github.io/js-lingui/guides/excluding-build-files
```

Comment:
Hmmm

Reply:
O joy the good old capitalization issue. 

If you removed the dile try to revert it to the parent (original) version in a separate commit. That usually works.

Reply:
We did it 😄